velox/functions/prestosql/ArrayIntersectExcept.cpp

```diff
@@ -65,6 +62,22 @@ void generateSet(
   }
 }
 
+DecodedVector* decodeArrayElements(
+    exec::LocalDecodedVector& arrayDecoder,
```

Comment:
Can this argument be const ?

Reply:
The arrayDecoder.get() call and elementsDecoder.get() calls are not const functions, so can't make them const.